src/gui/transferlistmodel.h

```diff
@@ -39,6 +40,47 @@ namespace BitTorrent
     class TorrentHandle;
 }
 
+// convenience class to read text colors from qss since qss doesn't work with qobject
+class TransferListTextColors : public QWidget
```

Comment:
Either
1. rename it to TransferListColors and drop "Color" part from property names, or 
2. drop "TextColor" part from property names.

Reply:
The first one will create clash with `TransferList` so I chose the latter.

Reply:
I chose the latter. done?